go/cmd/deploy/main.go

```diff
@@ -119,55 +119,84 @@ var DeployFlags = []cli.Flag{
 	cli.String("auth-token", "Control plane auth token", cli.Default(DefaultAuthToken)),
 }
 
+// WARNING: Changing the "Description" part will also affect generated MDX.
 // Cmd defines the deploy CLI command
 var Cmd = &cli.Command{
 	Name:  "deploy",
 	Usage: "Deploy a new version or initialize configuration",
 	Description: `Build and deploy a new version of your application, or initialize configuration.
 
-When used with --init, creates a configuration template file.
-Otherwise, builds a container image from the specified context and
-deploys it to the Unkey platform.
-
-The deploy command will automatically load configuration from unkey.json
-in the current directory or specified config directory.
-
-EXAMPLES:
-    # Initialize configuration file
-    unkey deploy --init
-
-    # Initialize in specific directory
-    unkey deploy --init --config=./my-project
-
-    # Force overwrite existing config
-    unkey deploy --init --force
-
-    # Deploy using config file (./unkey.json)
-    unkey deploy
-
-    # Deploy with config from specific directory
-    unkey deploy --config=./test-docker
-
-    # Deploy overriding workspace from config
-    unkey deploy --workspace-id=ws_different
-
-    # Deploy with specific context (overrides config)
-    unkey deploy --context=./demo_api
-
-    # Deploy with your own registry
-    unkey deploy \
-      --workspace-id=ws_4QgQsKsKfdm3nGeC \
-      --project-id=proj_9aiaks2dzl6mcywnxjf \
-      --registry=docker.io/mycompany/myapp
-
-    # Local development (skip push)
-    unkey deploy --skip-push
-
-    # Deploy pre-built image
-    unkey deploy --docker-image=ghcr.io/user/app:v1.0.0
-
-    # Show detailed build and deployment output
-    unkey deploy --verbose`,
+The deploy command handles the complete deployment lifecycle: from building Docker images to deploying them on Unkey's infrastructure. It automatically detects your Git context, builds containers, and manages the deployment process with real-time status updates.
+
+**Initialization Mode:**
+Use --init to create a configuration template file. This generates an unkey.json file with your project settings, making future deployments simpler and more consistent across environments.
+
+**Deployment Process:**
+1. Load configuration from unkey.json or flags
+2. Build Docker image from your application
+3. Push image to container registry
+4. Create deployment version on Unkey platform
+5. Monitor deployment status until active
+
+## Examples
+
+#### Initialize new project configuration
+` + "```bash" + `
```

Comment:
I would probably just indent this or something to visualise it instead of this ugly escaping 
cause it'll probably look quite weird on the terminal too, no?

Reply:
Btw, we can add a unique symbol like `-->`, so when parsing we can assume those are codeblocks. If folks prefer that ofc 😄

Reply:
I don't have a strong opinion, but @chronark might be right. If you fuck up the ticks(`) go won't compile, but if you miss that special char everything will be compiled and potentially will produce faulty docs